config/acme/machines/Makefile

```diff
@@ -351,6 +346,7 @@ else
   endif
 endif
 
+COMP_INTERFACE := $(shell echo $(COMP_INTERFACE) | tr A-Z a-z)
```

Comment:
COMP_INTERFACE is passed in uppercase, but needs to be lower case in the include path